distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -480,6 +486,256 @@ public void testBasicRestAccess() throws Exception {
         getSecurityPolicy().waitForGuestAuthReady(url);
     }
 
+    @Test(expected = NoHttpResponseException.class)
+    public void testBasicAuthOverHttp() throws Exception {
+        String url = SERVICE_ROOT.getUrl()
+                .replace("https", "http") + "/catalog/query?q=*&src=local";
+
+        configureRestForBasic(SDK_SOAP_CONTEXT);
+
+        waitForSecurityHandlers(url);
+
+        //try basic auth over http
+        given().auth()
```

Comment:
Should probably put this section in a try and then configure rest for guest in a finally so that a failure from one test doesn't cascade the rest.